packages/ibm-products/src/components/Datagrid/Datagrid/addons/InlineEdit/InlineEditCell/InlineEditCell.js

```diff
@@ -401,9 +410,16 @@ export const InlineEditCell = ({
       <DatePicker
         {...datePickerPreparedProps}
         appendTo={outerButtonElement?.current?.parentElement}
-        ref={datePickerRef}
-        style={{
-          width: cell.column.totalWidth,
+        ref={(el) => {
+          datePickerRef.current = el;
+          if (datePickerRef.current && datePickerRef.current.style) {
```

Comment:
why do we need to assign `el` to `datePickerRef.current` ?
Can't we just use `el` here?
`el.style.width=`${cell.column.totalWidth}px`;`